styleguide/components/01-mixins/breakpoints/_breakpoints.scss

```diff
@@ -1,17 +1,27 @@
 @import "breakpoint";
 
 //
-// Tablet breakpoint (see $bp-tablet defined in _vars.scss).
+// Mobile breakpoint (see $bp-mobile defined in _vars.scss).
 // This asumes a min-width as defined by the variable.
 //
-@mixin tablet {
-  @include breakpoint($bp-tablet) {
+@mixin mobile {
+  @include breakpoint($bp-mobile) {
     @content;
   }
 }
 
-@mixin tablet-only {
-  @include breakpoint($bp-tablet $bp-desktop) {
+@mixin mobile-only {
+  @include breakpoint($bp-mobile $bp-tablet) {
```

Comment:
0 to $bp-tablet, also not a fan of this approach, it's beter to avoid this use case.

Reply:
Agreed, we'll reomve the mixin-only() mixins.